src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,33 @@
+
+<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="codeu.model.data.Message" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Admin</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <!--
```

Comment:
Delete this block of commented out code. Keep the code clean. You can always add it back in later if you need it.